docs/source/best-practices/checkpointing.rst

```diff
@@ -74,8 +86,53 @@ To submit this array job to the scheduler, use the following command::
 
    sbatch submit_mdrun_array.bash
 
-Python TensorFlow checkpointing example
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Checkpointing using DMTCP
+--------------------------
+
+`DMTCP <https://dmtcp.sourceforge.io/>`_ (Distributed MultiThreaded checkpointing) is a tool that lets you checkpoint without the need to change your code. It Works with most Linux applications such as Python, Matlab, R, GUI, MPI, etc.
```

Comment:
[DMTCP](https://dmtcp.sourceforge.io/) (Distributed MultiThreaded checkpointing) is available on the cluster, allows checkpointing without the need to change your code, and works with most Linux applications (e.g., Python, Matlab, R, GUI, and MPI). DMTCP runs in the background of your program without causing significant performance loss and saves the process states into checkpoint files.